doc/languages-frameworks/emscripten.section.md

```diff
@@ -4,179 +4,51 @@
 
 This section of the manual covers how to use `emscripten` in nixpkgs.
 
-Minimal requirements:
+## Declarative usage (with `mkDerivation`) {#emscripten-declarative-usage}
 
-* nix
-* nixpkgs
+This mode is far more power full since this makes use of `nix` for
```

Comment:
```suggestion
This mode is far more powerful since this makes use of `nix` for
```

Though this sentence doesn't really make sense: More powerful than what? Imperative Usage is only described later on…